.env_file

```diff
@@ -14,3 +14,9 @@ MON_PORT=8765
 YDB_ENDPOINT=grpc://localhost:2136
 YDB_DATABASE=/local
 YDB_ANONYMOUS_CREDENTIALS=1
+CLICKHOUSE_DB=test
+CLICKHOUSE_USER=username
+CLICKHOUSE_DEFAULT_ACCESS_MANAGEMENT=1
+CLICKHOUSE_PASSWORD=pass
+SUPERSET_USERNAME=superset
+SUPERSET_PASSWORD=superset
```

Comment:
How to login into superset? I've tried `superset:superset` on `http://localhost:8088/login/` after running `docker-compose up` but I got `Invalid login`.

Reply:
Password and login are configured in the entrypoint script, so it takes from 0.5 to 1.5 minutes for these settings to be applied. It may be that you tried too early.